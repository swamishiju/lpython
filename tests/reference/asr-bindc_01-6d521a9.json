{
    "basename": "asr-bindc_01-6d521a9",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/bindc_01.py",
    "infile_hash": "3cfb601d3294c470842a85777832f5582ab52cb5bd64c0e02d40deb6",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-bindc_01-6d521a9.stderr",
    "stderr_hash": "966ff6032c983c2d80535d8c75db8cf87e1896d40a321a39f7f634c3",
    "returncode": 2
}