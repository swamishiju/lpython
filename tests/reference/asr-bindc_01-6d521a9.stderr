semantic error: Function 'empty_c_void_p' is not declared and not intrinsic
 --> tests/../integration_tests/bindc_01.py:3:17
  |
3 | queries: CPtr = empty_c_void_p()
  |                 ^^^^^^^^^^^^^^^^ 
